create-neon/data/templates/README.md.hbs

```diff
@@ -1,5 +1,82 @@
 # {{package.name}}
 
 {{#if package.description}}
-{{package.description}}
+*{{package.name}}:* {{package.description}}
+
 {{/if}}
+This project was bootstrapped by [create-neon](https://www.npmjs.com/package/create-neon).
+
+## Building {{package.name}}
+
+Building {{package.name}} requires a [supported version of Node and Rust](https://github.com/neon-bindings/neon#platform-support).
+
+You can build the project with npm. In the project directory, run:
+
+```sh
+$ npm install
```

Comment:
I would call this installing and not building. Building can be documented as `npm run build`. In many cases, users will want to run `npm run build` instead of `npm install` because it will be more efficient. `npm install` needs to check the dependency tree and potentially install updates.